uwblueprint/extend-a-family
# Remove google login

## Notion ticket link

[Ticket Name](https://www.notion.so/uwblueprintexecs/Remove-Google-Login-2381bf2299c14678b364abd900965ddd)



## Implementation description
* We shouldn’t have the “Login with Google” button on the login page anymore or any other instances of the Google login in general.



## Steps to test
1. Make sure the localhost website has no places with a "Log in with Google" type option.
2. If you want, check the branch to ensure there aren't any instances that refer to logging in with Google.


## What should reviewers focus on?
* Make sure you can't log in with Google doesn't exist as an option.


## Checklist
